Integrations/src/main/java/io/deephaven/integrations/numpy/Java2NumpyCopy.java

```diff
@@ -6,6 +6,7 @@
 import io.deephaven.db.v2.sources.ColumnSource;
```

Comment:
After reviewing this file, the changes here need to be in multiple PRs.
1) PR to fix the broken tests
2) PR that changes sampling algo
3) PR to adds methods to populate numpy arrays from column sources -- possibly belongs in this PR.

Reply:
1: Seems that this is the reason that `build.gradle` had `java.srcDirs = []` rather than what I have now, `java.srcDirs = [src/test/java]`. Before me adding all the _learn_ testing, I believe `Java2NumpyCopy` was the only thing being tested there, and it continues to fail on account of `"Could not initialize class io.deephaven.dbtypes.FactoryInstances"`. So, the simplest fix is to just not run those tests, which I guess is they weren't running them when I got here.

2: Cool.

3. These methods were commented out because not only did they not work, but the only that was changed about them from the code that I copied were the signatures; no other work had really been done. They were also started before the redesign, I haven't thought about if they'd still apply. I haven't looked at them in a while and I don't think they're integral for alpha version of this library. Maybe a future addition, but I see no reason to keep them around for now.

Reply:
1) Tests need to run.  You said you had fixed them.
3) Then delete them if they are garbage.

Reply:
Your code doesn't compile, so your tests don't run.  You should be able to run the tests locally through IntelliJ.  Then you can resolve problems before it gets into CI.

![image](https://user-images.githubusercontent.com/5250374/129104512-75f05a34-5209-4e26-b56f-f3560fa3e0d7.png)
![image](https://user-images.githubusercontent.com/5250374/129104616-1ac3aac0-0c2f-455f-998c-905a035811b4.png)

Reply:
You can also run the tests locally via gradle